src/control/cmd/dmg/pool_test.go

```diff
@@ -226,7 +226,7 @@ func TestPoolCommands(t *testing.T) {
 			"pool exclude 031bcaf8-f0f5-42ef-b3c5-ee048676dceb --rank 0 --target-idx 1",
 			strings.Join([]string{
 				printRequest(t, &control.PoolExcludeReq{
-					UUID:      "031bcaf8-f0f5-42ef-b3c5-ee048676dceb",
+					ID:        "031bcaf8-f0f5-42ef-b3c5-ee048676dceb",
```

Comment:
I don't see it blocking this PR but do we want to extend all of these tests in the future to also be based on label as well as UUID since I believe the purpose of doing this is to support labels in pool commands?